Aquality.Appium.Mobile.Template/Aquality.Appium.Mobile.Template/Resources/settings.json

```diff
@@ -0,0 +1,51 @@
+﻿{
+  "platformName": "android",
+  "assemblyNameWithScreens": "Aquality.Appium.Mobile.Template",
+  "isRemote": true,
+  "remoteConnectionUrl": "http://127.0.0.1:4723/wd/hub",
+
+  "driverSettings": {
+    "android": {
+      "applicationPath": "./Resources/Applications/TheApp-v1.10.0.apk",
+      "capabilities": {
+        "platformName": "Android",
+        "deviceName": "Android Emulator",
+        "automationName": "UiAutomator2",
+        "appPackage": "io.cloudgrey.the_app",
+        "appActivity": "io.cloudgrey.the_app.MainActivity"
```

Comment:
do we need to specify both appPackage/activity and applicationPath?

Reply:
Yes

Reply:
why? app would be started even if you would not specify the activity. Does it start with a different activity?